x11-misc/libinput-gestures/libinput-gestures-9999.ebuild

```diff
@@ -26,7 +30,17 @@ DEPEND=">=dev-libs/libinput-1.8.0
 	dev-util/desktop-file-utils
 	test? ( dev-python/flake8[${PYTHON_USEDEP}] )"
 
+src_prepare() {
+	default
+
+	# Fix docdir installation path
+	sed '/^DOCDIR/s@$NAME@${PF}@' -i libinput-gestures-setup || die
```

Comment:
Why do we need this?
PROG will have a value libinput-gestures-setup, and NAME - libinput-gestures, which is the same with ${PN}.

Reply:
Because according to the [Gentoo devmanual](https://devmanual.gentoo.org/function-reference/install-functions/index.html) docs have to go into `/usr/share/doc/${PF}` (see `dodoc` function).

Right now the ebuild installs the **same** docs into these two directories:
`/usr/share/doc/${PN}` and `/usr/share/doc/${PF}`

Reply:
I didn't notice that you were talking about ${PF}, not ${PN}.
Agreed.